client/src/components/authentication/SignInForm.js

```diff
@@ -64,22 +64,33 @@ const SignInForm = () => {
   }
 
   return (
+    <div className="form">
+    <div className="trail-bg-img-signin">
     <div className="grid-container" onSubmit={onSubmit}>
```

Comment:
These `div`'s should be indented and nested under one another. You can also combine CSS class names onto the same property. If these 3 classes:
- form
- trail-bg-img-signin
- grid-container

are _all_ for the entire SignInForm, they can be written as one class
```
className="form trail-bg-img-signin grid-container"
```
Which means you can remove the extra `div`'s and are left with the one surrounding the form

(You'll also want to move the `onSubmit` to the `form` element, this is an error from engage)

Reply:
wasn't too too bad